content/references/protocol-reference/transactions/transaction-types/xchainclaim.md

```diff
@@ -0,0 +1,65 @@
+---
+html: xchainclaim.html 
+parent: transaction-types.html
+blurb: Create a bridge between two chains.
+labels:
+  - Interoperability
+status: not_enabled
+---
+# XChainClaim
+
+The `XChainClaim` transaction completes a cross-chain transfer of value. It allows a user to claim the value on the destination chain - the equivalent of the value locked on the source chain. A user can only claim the value if they own the cross-chain claim ID associated with the value locked on the source chain (the `Account` field). The user can send the funds to anyone (the `Destination` field). This transaction is only needed if an `OtherChainDestination` is not specified in the XChainCommit transaction, or if something goes wrong with the automatic transfer of funds.
+
+
+## Example {{currentpage.name}} JSON
+
+
+```json
+{
+  "Account": "rahDmoXrtPdh7sUdrPjini3gcnTVYjbjjw",
+  "Amount": "10000",
+  "TransactionType": "XChainClaim",
+  "XChainClaimID": "13f",
+  "Destination": "rahDmoXrtPdh7sUdrPjini3gcnTVYjbjjw",
+  "XChainBridge": {
+    "LockingChainDoor": "rMAXACCrp3Y8PpswXcg3bKggHX76V3F8M4",
+    "LockingChainIssue": {
+      "currency": "XRP"
+    },
+    "IssuingChainDoor": "rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh",
+    "IssuingChainIssue": {
+      "currency": "XRP"
+    }
+  }
+}
+```
+
+
+{% include '_snippets/tx-fields-intro.md' %}
+
+| Field         | JSON Type           | [Internal Type][] | Description        |
+|:--------------|:--------------------|:------------------|:-------------------|
+| `XChainBridge`| String | Object | _Required_ The XChainBridge stanza represents the bridge for which the witness is attesting transactions. |
```

Comment:
This is not a string in JSON. Also, I recommend adding a "Required?" column to this table as well.


```suggestion
| `XChainBridge` | Object | STObject | Yes | The bridge for which the witness is attesting transactions. |
```